lib/ast-utils.js

```diff
@@ -158,9 +158,9 @@ function isCallee(node) {
 }
 
 /**
- * Checks whether or not a node is `Reclect.apply`.
+ * Checks whether or not a node is `Reflect.apply`.
```

Comment:
These fixes should be removed from here, since we've already merged your other PR that fixes these